sdk/workloads/Azure.ResourceManager.Workloads/src/Generated/Models/RedisCacheFamily.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.Workloads.Models
+{
+    /// <summary> Cache family. </summary>
+    public readonly partial struct RedisCacheFamily : IEquatable<RedisCacheFamily>
+    {
+        private readonly string _value;
+
+        /// <summary> Initializes a new instance of <see cref="RedisCacheFamily"/>. </summary>
+        /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
+        public RedisCacheFamily(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        private const string CValue = "C";
+        private const string PValue = "P";
+
+        /// <summary> C. </summary>
+        public static RedisCacheFamily C { get; } = new RedisCacheFamily(CValue);
+        /// <summary> P. </summary>
+        public static RedisCacheFamily P { get; } = new RedisCacheFamily(PValue);
```

Comment:
It's weird to use single letter, can we change it?

Reply:
It's Redis concept, I don't think we need to rename it